apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -176,10 +168,10 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
   }, []);
 
   const getScores = useCallback(
-    (useInputs: string[], step: number) => {
+    (myInputs: string[], step: number) => {
```

Comment:
`useInputs` as a variable looks like a hook, which it isn't.

changed to `myInputs` to still be local, but not conflicting.

Reply:
I don't think we talk about `my` in most variables.. how about `currentInputs`?

Reply:
I want that variable to emphasize that it's local to those functions and not state in the modal. `currentInputs` still feels ambiguous (especially with the other discussion about `previousInputs` vs `currentInputs`). A `my` prefix emphasizes it nicely.

What about `givenInputs`?